lib/matplotlib/_constrained_layout.py

```diff
@@ -248,6 +261,42 @@ def check_no_collapsed_axes(layoutgrids, fig):
     return True
 
 
+def compress_fixed_aspect(layoutgrids, fig):
+    gs = None
+    for ax in fig.axes:
+        if hasattr(ax, 'get_subplotspec'):
```

Comment:
I suppose this could also flip to `if not hasattr(ax, 'get_subplotspec'): continue` as well.

Reply:
Done.  At some point we were taught that "break" and "continue" are not desirable.